m4/japhar_grep_cflags.m4

```diff
@@ -31,12 +31,9 @@ foo (void)
 {
   exit(1);
 }
-],
-[
+]], [[
   exit (0);
-],
-ac_cv___attribute__=yes,
-ac_cv___attribute__=no)])
+]])],[ac_cv___attribute__=yes],[ac_cv___attribute__=no])])
```

Comment:
The old formatting is easier to read than the new one.
Could you keep the old formatting, please?